plugins/woocommerce/client/blocks/assets/js/blocks/mini-cart/iapi-frontend.ts

```diff
@@ -164,3 +170,145 @@ store(
 	},
 	{ lock: true }
 );
+
+type CartItemContext = {
+	cartItem: CartItem;
+};
+
+const { state } = store( 'woocommerce/mini-cart-items-block', {
+	state: {
+		get cartItem() {
+			return getContext< CartItemContext >( 'woocommerce' ).cartItem;
+		},
+
+		// Intended to be used in context of a cart item in wp-each
+		get reduceQuantityLabel(): string {
+			const { reduceQuantityLabel } = getConfig(
+				'woocommerce/mini-cart-items-block'
+			);
```

Comment:
I was thinking. Maybe it's not necessary to execute the `getConfig` every time a getter is executed. Since these values don't change, they could be moved out of the store.

Reply:
Good point. For some reason I assumed scope was necessary, just lack of knowledge of the API. its a good change 👍🏻